has_scope.gemspec

```diff
@@ -24,6 +24,9 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.1.7'
 
-  s.add_runtime_dependency 'actionpack', '>= 4.1', '< 5.2'
-  s.add_runtime_dependency 'activesupport', '>= 4.1', '< 5.2'
+  s.add_runtime_dependency 'actionpack', '>= 4.1', '< 5.3'
```

Comment:
As a nice to have, I think '<= 5.2.x' works with 5.2.beta, which is a bit more elegant, as the next release is probably 6.0.